README.md

```diff
@@ -28,42 +41,42 @@ There are several ways to install this app onto a workload cluster.
 
 ```yaml
 # values.yaml
-
+global:
+  remoteWrite:
+  - name: "remotewrite" 
+    url: "http://$BASEDOMAIN/write"
+
+prometheus-agent:
+  serviceMonitor:
+    enabled: true
+    relabelings:
+    # Add app label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_name
+      targetLabel: app
+    # Add instance label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_app_kubernetes_io_instance
+      targetLabel: instance
+    # Add node label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_node_name
+      targetLabel: node
+    # Add team label.
+    - sourceLabels:
+      - __meta_kubernetes_pod_label_application_giantswarm_io_team
```

Comment:
This label look quite giantswarm-specific.
A customer who would install a prometheus-agent to scrape his own services/pods would probably not care about the giantswarm team labels.

Reply:
I'd remove it yes